Description: Add support for complete Debian hardening
 Support for including complementary CFLAGS from
 debian/rules.
Author: Philippe Thierry
Last-Update: 2017-05-23
--- a/Makefile
+++ b/Makefile
@@ -7,11 +7,11 @@
 # This file is licensed under MIT License.
 #
 
-CFLAGS := -W -Wall -Wno-unused-parameter -Wno-unused-function -Wno-unused-result -fpie -pie -fPIC -g3 -ggdb -I../../include  -I./include/sflib/ -I./include -I../../include/  -Wno-incompatible-pointer-types  -fstack-protector-all -Wl,-z,relro,-z,now -DPACKAGE -DPACKAGE_VERSION -masm=intel -rdynamic -D_fORTIFY_SOURCE=2 -O2
+CFLAGS += -W -Wall -Wno-unused-parameter -Wno-unused-function -Wno-unused-result -fpie -pie -fPIC -g3 -ggdb -I../../include  -I./include/sflib/ -I./include -I../../include/  -Wno-incompatible-pointer-types  -fstack-protector-all -Wl,-z,relro,-z,now -DPACKAGE -DPACKAGE_VERSION -masm=intel -rdynamic -D_fORTIFY_SOURCE=2 -O2
 
 all:
 	mkdir -p bin
-	cd src && make CFLAGS=" $(CFLAGS)"
+	cd src && make CFLAGS="$(CPPFLAGS) $(CFLAGS)"
 
 documentation:
 	cd src && doxygen ./tex/project.cfg
